Sources/MessagingInApp/Views/UIKitInline.swift

```diff
@@ -100,6 +70,22 @@ public class InAppMessageView: UIView {
     }
 
     private func setupView() {
+        guard let elementId, inAppMessageView == nil else {
+            return // We are already setup. No need to do again.
+        }
+
+        let inlineInAppMessageView = GistInlineInAppMessageView(elementId: elementId)
+        inlineInAppMessageView.delegate = self
+        addSubview(inlineInAppMessageView)
+
+        inlineInAppMessageView.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            inlineInAppMessageView.centerXAnchor.constraint(equalTo: centerXAnchor),
+            inlineInAppMessageView.centerYAnchor.constraint(equalTo: centerYAnchor),
+            inlineInAppMessageView.widthAnchor.constraint(equalTo: widthAnchor),
+            inlineInAppMessageView.heightAnchor.constraint(equalTo: heightAnchor)
+        ])
```

Comment:
Add the base `GistInlineInAppMessageView` UIKit view as a subview, set the `delegate`, and then all other logic in this View comes from the new delegate functions.